nusc_exp_mgr_node:
  ros__parameters:
    exp_configs: ['e0_ab3dmot_baseline']
    val_dataset: 'v1.0-mini'
    val_split: 'mini_val'
    mcap_dir: 'nuscenes/data' # relative to home directory
    results_dir: 'tracking_ws/src/ros_tracking/results' # relative to home directory